doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -1457,7 +1491,7 @@ NCSDK-20457: Modem traces captured through UART are corrupted if RTT logs are si
 
   This increases the overall power consumption even when Serial LTE Modem is in sleep mode.
 
-.. rst-class:: v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0
+.. rst-class:: v2-7-0 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0
```

Comment:
Only present in v2-5-0. I believe that this and the 23733 one have gotten mix up. Apologies.

```suggestion
.. rst-class:: v2-5-0
```

Reply:
for: NCSDK-23733: Full modem firmware update issue on the nRF9161 DK

rst-class:: v2-7-0 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0

 **Affected platforms:** nRF9161, nRF9151